hbase-server/src/main/resources/hbase-webapps/master/table.jsp

```diff
@@ -834,12 +844,19 @@ if (withReplica) {
 </tbody>
 </table>
 <% }
-} catch(Exception ex) {
+} catch(Exception ex) { %>
+  Unknown Issue with Regions
+  <div onclick="document.getElementById('closeRegion').style.display='block';document.getElementById('openRegion').style.display='none';">
+    <a id="openRegion" style="cursor:pointer;"> Show StackTrace</a>
+  </div>
+  <div id="closeRegion" style="display:none;clear:both;">
+    <div onclick="document.getElementById('closeRegion').style.display='none';document.getElementById('openRegion').style.display='block';">
+      <a style="cursor:pointer;"> Close StackTrace</a>
+    </div>
+  <%
   for(StackTraceElement element : ex.getStackTrace()) {
-    %><%= StringEscapeUtils.escapeHtml4(element.toString()) %><%
+    %><%= StringEscapeUtils.escapeHtml4(element.toString() + "\n") %><%
   }
-} finally {
```

Comment:
Removal of finally seems unintentional

Reply:
Previous commit was conflicted in code of the finally section.
The HBASE-23653 removed the section and closed connection by changing hbase-server/src/test/java/org/apache/hadoop/hbase/ConnectionRule.java.
I thought I should keep compatibility with the HBASE-23653.
How should I modify the code intentionally?

Reply:
Sure sounds good. Btw, QA run output says branch needs rebase with master. Could you please rebase one more time? Hopefully we should be able to merge after that.
Thanks

Reply:
Thank you for comment. I rebased HBASE-23578 with master.